alma/alma-installments-prestashop
# feat: add inpage setting for selector payment buttons

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-2080/[-ps]-refactor-in-page-to-fix-incompatibilities)

### Code changes


Add form settings for selector InPage payment buttons
The settings are get in the js inpage code and can be customized for compatibility with specific checkout

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Change the settings of in page selector.
And see if the settings are well selected.
For a better test, you can change the attribute needed for in-page to test with the specific selector if the in page work fine.

### Checklist for authors and reviewers




### Non applicable
